<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QVisualizerClass</class>
 <widget class="QMainWindow" name="QVisualizerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>512</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QVisualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTopics">
    <property name="title">
     <string>Topics</string>
    </property>
    <widget class="QMenu" name="menuScalar_Field_Visualization">
     <property name="title">
      <string>Scalar Field Visualization</string>
     </property>
     <addaction name="actionMarching_Cubes"/>
     <addaction name="actionVolume_Rendering"/>
    </widget>
    <addaction name="menuScalar_Field_Visualization"/>
    <addaction name="actionVector_Field_Visualization"/>
    <addaction name="actionTensor_Field_Visualization"/>
    <addaction name="actionVirtual_Environment_For_Visualization"/>
    <addaction name="actionLarge_Scale_Data_Visualization"/>
    <addaction name="actionVisualization_Software_and_Frameworks"/>
    <addaction name="actionPerceptual_Issues_in_Visualization"/>
    <addaction name="actionSelected_Topics_and_QApplications"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuTopics"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>41</height>
    </size>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionVector_Field_Visualization">
   <property name="text">
    <string>Vector Field Visualization</string>
   </property>
  </action>
  <action name="actionMarching_Cubes">
   <property name="text">
    <string>Marching Cubes</string>
   </property>
  </action>
  <action name="actionVolume_Rendering">
   <property name="text">
    <string>Volume Rendering</string>
   </property>
  </action>
  <action name="actionTensor_Field_Visualization">
   <property name="text">
    <string>Tensor Field Visualization</string>
   </property>
  </action>
  <action name="actionVirtual_Environment_For_Visualization">
   <property name="text">
    <string>Virtual Environment For Visualization</string>
   </property>
  </action>
  <action name="actionLarge_Scale_Data_Visualization">
   <property name="text">
    <string>Large-Scale Data Visualization</string>
   </property>
  </action>
  <action name="actionVisualization_Software_and_Frameworks">
   <property name="text">
    <string>Visualization Software and Frameworks</string>
   </property>
  </action>
  <action name="actionPerceptual_Issues_in_Visualization">
   <property name="text">
    <string>Perceptual Issues in Visualization</string>
   </property>
  </action>
  <action name="actionSelected_Topics_and_QApplications">
   <property name="text">
    <string>Selected Topics and QApplications</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
